pyproject.toml

```diff
@@ -1,6 +1,100 @@
+[build-system]
+build-backend = "hatchling.build"
+requires = [
+  "hatchling",
+]
+
+[project]
+name = "rq"
+description = "RQ is a simple, lightweight, library for creating background jobs, and processing them."
+readme = "README.md"
+license = "BSD-2-Clause"
+maintainers = [
+    {name = "Selwin Ong"},
+]
+authors = [
+    { name = "Vincent Driessen", email = "vincent@3rdcloud.com" },
+]
+requires-python = ">=3.6.2"
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Intended Audience :: Developers",
+  "Intended Audience :: End Users/Desktop",
+  "Intended Audience :: Information Technology",
+  "Intended Audience :: Science/Research",
+  "Intended Audience :: System Administrators",
+  "License :: OSI Approved :: BSD License",
+  "Operating System :: MacOS",
+  "Operating System :: POSIX",
+  "Operating System :: Unix",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.6",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Topic :: Internet",
+  "Topic :: Scientific/Engineering",
+  "Topic :: Software Development :: Libraries :: Python Modules",
+  "Topic :: System :: Distributed Computing",
+  "Topic :: System :: Monitoring",
+  "Topic :: System :: Systems Administration",
+]
+dynamic = [
+  "version",
+]
+dependencies = [
+  "click>=5",
+  "redis>=3.5",
+]
+[project.urls]
+changelog = "https://github.com/rq/rq/blob/master/CHANGES.md"
+documentation = "https://python-rq.org/docs/"
+homepage = "https://python-rq.org/"
+repository = "https://github.com/rq/rq/"
+[project.scripts]
+rq = "rq.cli:main"
+rqinfo = "rq.cli:info"  # TODO [v2]: Remove
+rqworker = "rq.cli:worker"  # TODO [v2]: Remove
+
+[tool.hatch.version]
+path = "rq/version.py"
+
+[tool.hatch.build.targets.sdist]
+include = [
+    "/docs",
+    "/rq",
+    "/tests",
+    "CHANGES.md",
+    "LICENSE",
+    "pyproject.toml",
+    "README.md",
+    "requirements.txt",
+    "tox.ini",
+]
+
+[tool.hatch.envs.test]
+dependencies = [
+  "black",
+  'coverage; python_version >= "3.7"',
+  'coverage==6.2; python_version == "3.6"',
```

Comment:
We no longer need to support Python 3.6.

Reply:
Okay. I can do another pass to make sure I remove this plus some others I see.